nixops/resources/__init__.py

```diff
@@ -189,6 +189,16 @@ def destroy(self, wipe=False):
         )
         return False
 
+    def clear_state(self):
+        """clear this resource state, if possible."""
+        if not self.depl.logger.confirm("are you sure you want to clear resources state {}: this will only remove the resource from the local NixOPS state and the resource may still exist outside of the NixOPS database.".format(self.name)):
```

Comment:
Can you split into multiple lines, something like:
```python
def clear_state(self):
    """clear this resource state, if possible."""
    if not self.depl.logger.confirm(
            "are you sure you want to clear resources state {}:"
            "this will only remove the resource from the local "
            "NixOPS state and the resource may still exist outside"
            " of the NixOPS database.".format(self.name)):
        return False
```